data/net/minecraft/nbt/CompoundTag.mapping

```diff
@@ -1,109 +1,99 @@
 CLASS net/minecraft/nbt/CompoundTag
+	METHOD <init> (Ljava/util/Map;)V
+		ARG 1 tags
 	METHOD contains (Ljava/lang/String;)Z
-		COMMENT Returns whether the given string has been previously stored as a key in the map.
 		ARG 1 key
 	METHOD contains (Ljava/lang/String;I)Z
-		COMMENT Returns whether the given string has been previously stored as a key in this tag compound as a particular type, denoted by a parameter in the form of an ordinal. If the provided ordinal is 99, this method will match tag types representing numbers.
+		COMMENT Returns whether the tag of the specified {@code key} is a
+		COMMENT particular {@code tagType}. If the {@code tagType} is {@code 99},
+		COMMENT all numeric tags will be checked against the type of the stored tag.
```

Comment:
Soooo... standards decision on whether to include intentional linebreaks in javadoc comments for any given arbitrary column width? If I remember correctly we had said to only add specific line breaks when one was actually desired and leave the splitting of words up to the mapping consumer (as this does not account for things like indentation of the original javadoc, etc.)

In any case, I'd think this is all one line, or two (one per sentence), no?

Reply:
I forgot about this. Technically it should all be on one line or on multiple lines. I was just formatting so I could read while I was mapping. I'll fix this.